ionic2/app/pages/wiki-index/wiki-index.page.ts

```diff
@@ -12,17 +12,22 @@ import { ApiService } from '../../services/api.service';
   directives: [forwardRef(() => NavbarComponent)]
 })
 
-export class WikiIndexPage implements OnInit, OnDestroy {
+export class WikiIndexPage implements OnDestroy {
 
   queryString: string;
   querySubscription: Subscription;
   results: Pokemon[] = [];
 
   constructor(private navCtrl: NavController, private apiservice: ApiService) { }
 
-  ngOnInit() {
-    this.querySubscription = this.apiservice.getAllPokemon()
+  ionViewDidEnter() {
```

Comment:
Is your change already live? Didn't display a merge conflict with ionViewDidEnter